mittab/apps/tab/models.py

```diff
@@ -38,43 +54,65 @@ def delete(self):
         if len(team_check) == 0 and len(judge_check) == 0:
             super(School, self).delete()
         else:
-            raise Exception("School in use: [teams => %s,judges => %s]" % ([t.name for t in team_check], [j.name for j in judge_check]))
+            raise Exception("School in use: [teams => %s,judges => %s]" % (
+                [t.name for t in team_check], [j.name for j in judge_check]))
 
 
 class Debater(models.Model):
-    name = models.CharField(max_length=30, unique = True)
-    #team_set is created by Team in the ManyToMany
-    #team = models.ForeignKey('Team')
-    #0 = Varsity, 1 = Novice
+    """
+    Contains information with regard to the debaters at the tournament.
+    """
+    # TODO it's honestly probably better to make a super-class containing the provider, phone, and name information.
+    # Then, make the Judge and the Debater classes extend that class.
+    name = models.CharField(max_length=30, unique=True)
+    """ name is the name of the Debater """
+
+    # team_set is created by Team in the ManyToMany
+    # team = models.ForeignKey('Team')
+    # 0 = Varsity, 1 = Novice
     VARSITY = 0
     NOVICE = 1
     NOVICE_CHOICES = (
         (VARSITY, u'Varsity'),
         (NOVICE, u'Novice'),
     )
+
     phone = PhoneNumberField(blank=True)
+    """ field for the debater's phone number """
+
     provider = models.CharField(max_length=40, blank=True)
+    """ field for the provider of the debater"""
+
     novice_status = models.IntegerField(choices=NOVICE_CHOICES)
+    """ field for whether the debater is a novice or not """
+
     def __unicode__(self):
         return self.name
 
     def delete(self):
-        teams = Team.objects.filter(debaters = self)
+        teams = Team.objects.filter(debaters=self)
         if len(teams) == 0:
             super(Debater, self).delete()
-        else :
+        else:
             raise Exception("Debater on teams: %s" % ([t.name for t in teams]))
 
+
 class Team(models.Model):
-    name = models.CharField(max_length=30, unique = True)
+    """
+    Contains information with regard to the team itself. Contains the team name, the team school, the debaters in that
+    team, the team seed, and the information on whetehr that team is checked in or not.
+    """
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z ]*$', 'Only alphanumeric characters are allowed.')
```

Comment:
I added this because if you try to use non-ASCII characters, Python complains when it is `str`-ed in `team_views.py`, preventing someone from adding the team.